mne/decoding/time_gen.py

```diff
@@ -215,12 +213,9 @@ def fit(self, epochs, y=None, picks=None):
         from sklearn.base import clone
         from sklearn.cross_validation import check_cv, StratifiedKFold
         n_jobs = self.n_jobs
-        # Default channel selection
-        if picks is None:
-            picks = pick_types(epochs.info, meg=True, eeg=True,
-                               exclude='bads')
         # Extract data from MNE structure
-        X, y = _check_epochs_input(epochs, y, picks)
+        self.ch_names = epochs.ch_names
+        X, y = _check_epochs_input(epochs, y)
```

Comment:
use copy.deepcopy from the standardlibrary, most explicit. Alternatively you can use list or make a list comprehension, less explicit though.